src/account.ts

```diff
@@ -123,7 +128,11 @@ export const isPrecompiled = function(address: Buffer | string): boolean {
  * Checks if the private key satisfies the rules of the curve secp256k1.
  */
 export const isValidPrivate = function(privateKey: Buffer): boolean {
-  return secp256k1.privateKeyVerify(privateKey)
+  try {
```

Comment:
I'm confused about why this version adds a `try/catch` while this other doesn't: https://github.com/ethereumjs/ethereumjs-util/pull/257/files#diff-49540b967120fc524b229469e9847e68R127

Reply:
Secp256k1 expects provided private key to be an Uint8Array with length 32. If this condition is not met the function will throw. I think this is the only case that can cause exception, I'll remove try/catch and add this check.

Reply:
Oh, I see. It must be a change in secp256k1 v4